includes/Form.class.php

```diff
@@ -165,6 +165,18 @@ function process($params) {
 		}
 		$email = $_POST["email"];
 		$phone = isset($_POST["phone"]) ? $_POST["phone"] : "";
+		$lists = (isset($_POST["nlbox"]) && $_POST["nlbox"]) ? $_POST["nlbox"] : array();
+		if (isset($_SESSION["image_random_value"])) {

```

Comment:
aah i see. so where is $_SESSION["image_random_value"] set then? i just find it odd that we process the form, and instead of checking the form settings we check the session. that means that crawling scripts that try to submit (have no session) will never enter this block.

we must be getting the form first, do we set that session var then, if the form has captcha in it? i dont see that in this file.